src/Orchard.Web/Modules/Orchard.OpenId/Controllers/AccessController.cs

```diff
@@ -19,22 +19,27 @@ public class AccessController : Controller
     {
         private readonly IOpenIdApplicationManager _applicationManager;
         private readonly SignInManager<User> _signInManager;
-        private readonly OpenIddict.OpenIddictUserManager<User> _userManager;
+        private readonly OpenIddictUserManager<User> _userManager;
 
         public AccessController(
             IOpenIdApplicationManager applicationManager,
             SignInManager<User> signInManager,
-            OpenIddict.OpenIddictUserManager<User> userManager)
+            OpenIddictUserManager<User> userManager)
         {
             _applicationManager = applicationManager;
             _signInManager = signInManager;
             _userManager = userManager;
         }
 
-        [HttpPost, Produces("application/json")]
-        public Task<IActionResult> Token()
+        [HttpPost, IgnoreAntiforgeryToken]
```

Comment:
Note: this change is not caused by the recent OpenIddict changes but is very likely needed to fix the indirect regression caused by https://github.com/OrchardCMS/Orchard2/commit/0a0a23b72dade941b759b077a234a2f78ecf47e8.